pset-functions.js

```diff
@@ -208,6 +314,22 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+function doubleLetters(word) {
+  const characters = word.split("")
+  let count = 0
+
+  for(let i = 0; i < characters.length; i++){
+      if(characters[i] == characters[i+1])
```

Comment:
@jacobkelman be careful with the syntax for the `if` statements

Including the curly braces improves the readability of your code

```
if (condition) {
}
```

Also, you should use `===` (strict comparison) instead of `==`